### DISCUSSION TITLE
Dual Spindle Questions

### DISCUSSION THREAD

**Discussion by rvalotta** [terjeio/grblHAL#351] [Q&A] [answered]
I'm looking for some understanding on how to get Dual spindle running.

I've got my VFD configured and had added the following 

#define SPINDLE_PWM_PORT_BASE   GPIOB_BASE
#define SPINDLE_PWM_PIN         3

in my board definition

I've tried $32=1 and nothing comes out of the PWM pin when I try and set it

**Comment by terjeio** [comment, marked as answer]
This version does not support dual spindle configuration, [the latest](https://github.com/grblHAL/STM32F4xx) does by adding
```
#if VFD_ENABLE
#undef VFD_SPINDLE 
#define VFD_SPINDLE 2
#endif
```
to your board map. I have to figure out how to enable dual spindle mode in _my_machine.h_ in a generic way...

And please, please do not start any new issues or discussions at this repo anymore - I want to archive it.
